docs/src/modules/components/MarkdownDocs.js

```diff
@@ -24,6 +37,63 @@ import AdManager from 'docs/src/modules/components/AdManager';
 import AdGuest from 'docs/src/modules/components/AdGuest';
 import ComponentLinkHeader from 'docs/src/modules/components/ComponentLinkHeader';
 
+const RATINGS_URL = 'https://5fm2imnpv2.execute-api.us-east-1.amazonaws.com/dev/rating';
+
+function Comment(props) {
+  const { onClose: handleClose, open } = props;
+  const t = useSelector((state) => state.options.t);
+  const [value, setValue] = React.useState('');
+
+  const handleChange = (event) => {
+    setValue(event.target.value);
+  };
+
+  const handleSubmit = () => {
+    handleClose(value);
+  };
+
+  const handleCancel = () => {
+    handleClose(null);
+  };
+
+  return (
+    <div>
+      <Dialog
+        open={open}
+        onChange={handleChange}
+        onClose={handleCancel}
+        aria-labelledby="form-dialog-title"
+        value={value}
```

Comment:
`onChange` and `value` should be on the TextField, no?

Reply:
Duh, yes!